package.json

```diff
@@ -46,37 +46,158 @@
     ],
     "permissionSets": [
       {
-        "permissionName": "module.finance.enabled",
-        "displayName": "UI: Finance module is enabled",
-        "visible": true
+        "permissionName": "ui-finance.fiscal-year.view",
+        "displayName": "Fiscal year: View",
+        "visible": true,
+        "subPermissions": [
+          "module.finance.enabled",
+          "acquisitions-units.units.collection.get",
+          "finance.budgets.collection.get",
+          "finance.fiscal-years.collection.get",
+          "finance.fiscal-years.item.get",
+          "finance.funds.collection.get",
+          "finance.group-fiscal-year-summaries.collection.get",
+          "finance.groups.collection.get",
+          "finance.ledgers.collection.get",
+          "users.collection.get"
+        ]
+      },
+      {
+        "permissionName": "ui-finance.fiscal-year.edit",
+        "displayName": "Fiscal year: View, edit",
+        "visible": true,
+        "subPermissions": [
+          "ui-finance.fiscal-year.view",
+          "acquisitions-units.memberships.collection.get",
+          "finance.fiscal-years.item.put"
+        ]
+      },
+      {
+        "permissionName": "ui-finance.fiscal-year.create",
+        "displayName": "Fiscal year: View, edit, create",
+        "visible": true,
+        "subPermissions": [
+          "ui-finance.fiscal-year.edit",
+          "finance.fiscal-years.item.post"
+        ]
+      },
+      {
+        "permissionName": "ui-finance.fiscal-year.delete",
+        "displayName": "Fiscal year: View, edit, delete",
+        "visible": true,
+        "subPermissions": [
+          "ui-finance.fiscal-year.edit",
+          "finance.fiscal-years.item.delete"
+        ]
       },
       {
-        "permissionName": "settings.finance.enabled",
```

Comment:
the same as above, I don't think it will work if you delete permission, but left it as a link in other permission sets

Reply:
but I don't use it anymore

Reply:
what do you mean by that? please read the permissions docs, https://dev.folio.org/faqs/explain-permissions-system/
especially https://github.com/folio-org/stripes-core/blob/master/doc/permissions.md#which-permissions-defined-where

Reply:
Reverted it and made it non-visible for users